src/components/calcite-filter/calcite-filter.tsx

```diff
@@ -42,6 +42,8 @@ export class CalciteFilter {
 
   @Element() el: HTMLCalciteFilterElement;
 
+  @State() empty = true;
```

Comment:
nitpick: Instead of `empty`, maybe `active` for when it does have a value populated?